cmd/root.go

```diff
@@ -135,6 +135,9 @@ func initConfig(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	// The value for `debug` might have changed; re-initialize the logger
+	initLogger()
+
```

Comment:
I really hate the whole initLogger() thing to begin with.  It's just goofy.  Calling initLogger at the end of `init()`, and the start of `initConfig()`, and now again in `initConfig()`.

I kinda feel like that's point to  a bad design in general on my part.

Reply:
True, but I like to think of myself as a one-problem-at-a-time kind of guy.  😆